src/plugins/intel_gpu/src/plugin/ops/eltwise.cpp

```diff
@@ -37,14 +37,17 @@ void CreateElementwiseOp(Program& p, const std::shared_ptr<ngraph::Node>& op, cl
     auto inputPrimitives = p.GetInputPrimitiveIDs(op);
     std::string layerName = layer_type_name_ID(op);
 
-    auto outRank = op->get_output_shape(0).size();
+    auto out_pshape = op->get_output_partial_shape(0);
+    auto out_rank = out_pshape.size();
     for (size_t i = 0; i < inputPrimitives.size(); ++i) {
```

Comment:
The entire loop of this can be applied only for static case.

Reply:
This is exactly what is checked below: input_pshape.is_static() && out_pshape.is_static()

Reply:
Yeah, so my question was, isnt it better to let the loop itself not to be entered if it is not static?

Reply:
I don't see any big difference in your approach

Reply:
Of course its not a critical change, but just for better readability and clearity to quickly check the static shape only part from the large point of view.

Reply:
updated